_data/people/irinaespejo.yml

```diff
@@ -34,3 +34,23 @@ presentations:
   meeting: Frontiers in Machine Learning for the Physical Sciences, UC Southern Hub
   meetingurl: https://sites.research.uci.edu/frontiers-machine-learning/symposium/
   location: Virtual
+- title: Active Learning for Levet Set Estimation
+  date: 2021-03-26
+  url: https://indico.fnal.gov/event/48483/
+  meeting: CLARIPHY seminar
+  meetingurl: https://clariphy.org/
+  location: Virtual
+- title: Active Learning for Level Set Estimation
+  date: 2021-03-30
+  url: https://www.google.com/url?q=https://indico.cern.ch/event/1019958/timetable/%2314-active-learning-for-level-s&sa=D&source=editors&ust=1631238621298000&usg=AOvVaw2XfVikLtLX4UYZVasr0G2l
```

Comment:
Can you put the actual URL, rather than the Google one?

Reply:
good catch, done :white_check_mark:

Reply:
Thanks!